lib/fastly_nsq/version.rb

```diff
@@ -1,3 +1,3 @@
 module FastlyNsq
-  VERSION = '0.6.0'.freeze
+  VERSION = '0.7.0'.freeze
```

Comment:
If this is breaking and non-backwards compatible, and changing the name you must use is exactly that, then the correct thing todo here would be to bump the `MAJOR` not the `MINOR`...